ecosystem/typescript/sdk_v2/src/api/typesafe.test.ts

```diff
@@ -0,0 +1,837 @@
+import { AptosConfig } from "./aptos_config";
```

Comment:
let's get this file under a `/src/tests/surf/` folder

Reply:
I wonder if we should still keep the name `surf`.

Reply:
moved

Reply:
> I wonder if we should still keep the name surf.
I agree, we can think about naming it later